notebooks/official/generative_ai/tune_peft.ipynb

```diff
@@ -0,0 +1,744 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # @title Copyright &amp; License (click to expand)
</pre><p>Remove this line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>